app/src/main/res/values/strings.xml

```diff
@@ -31,4 +31,21 @@
     <string name="logo">
     Logo
     </string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="note_item_desc_placholder">In this chapter, you’re going to learn about the different types of subjects in RxJava, see how to work with each one and why you might choose one over another based on some common use cases.</string>
```

Comment:
Ich weiß, da steht TODO dran und wird dann ja auch in AndroidStudio angezeigt, aber nur als kurzer Reminder

Reply:
Kann man erst mal vergessen